### ISSUE TITLE
 'NoneType' object has no attribute 'rfind'

### ISSUE THREAD

**Issue by telanus** [lutris/lutris#2455]
Was installing a game this morning, and found that it wouldn't, checked the logs and found this:

```
[wine.create_prefix:148]:win32 Prefix created in /Games/wine/xrx
ERROR    2019-11-12 10:26:19,038 [jobs.target:32]:Error while completing task <bound method wine.prelaunch of <lutris.runners.wine.wine object at 0x7f9acc965080>>: 'NoneType' object has no attribute 'rfind'
<class 'AttributeError'> 'NoneType' object has no attribute 'rfind'
  File "~/src/lutris/lutris/util/jobs.py", line 30, in target
    result = self.function(*args, **kwargs)
  File "~/src/lutris/lutris/runners/wine.py", line 787, in prelaunch
    create_prefix(self.prefix_path, arch=self.wine_arch)
  File "~/src/lutris/lutris/runners/commands/wine.py", line 150, in create_prefix
    prefix_manager.setup_defaults()
  File "~/src/lutris/lutris/util/wine/prefix.py", line 27, in setup_defaults
    self.desktop_integration()
  File "~/src/lutris/lutris/util/wine/prefix.py", line 91, in desktop_integration
    DESKTOP_FOLDERS.append(folder[folder.rfind("\\") + 1:])
ERROR    2019-11-12 10:26:19,106 [game.configure_game:312]:'NoneType' object has no attribute 'rfind'
Gtk-Message: 10:26:19.192: GtkDialog mapped without a transient parent. This is discouraged.
```

**Comment by tannisroot** [comment]
Which game?

**Comment by telanus** [comment]
@tannisroot I installed Xpand Rally Xtreme from CD. Was able to replicate it with Prince of Persia: The Sands of Time from GOG

Using latest wine-staging (4.20) .  

Steps to reproduce: add a new game to lutris, setup as wine game, select 32bit prefix. select a up-to-date wine-staging (since that's what I used) start game.

It'll go thru the motions of ceating a new prefix, and seems near the end of creating the prefix, or when the change happen between the prefix creation and stating the installer, it'll crash
